include/SKSE/Impl/PCH.h

```diff
@@ -499,7 +499,7 @@ namespace SKSE
 					fileview = fileview.substr(pos + off);
 				}
 
-				return fmt::format(FMT_STRING("{}({}): {}"), fileview, a_loc.line(), a_msg);
+				return fmt::format("{0}({1}): {2}", fileview, a_loc.line(), a_msg);
```

Comment:
Why would you remove compile time format checking and make the format more brittle to changes?